Quick_Deploy/vLLM/model_repository/vllm/1/model.py

```diff
@@ -123,26 +123,31 @@ def get_sampling_params_dict(self, params_json):
         This functions parses the dictionary values into their
         expected format.
         """
+        # Opinionated defaults for out-of-the-box experience if left unspecified.
```

Comment:
Assuming diverse variety of models can be loaded and served using this interface, how about reading defaults from config.pbtxt or any other file?

Reply:
I'll circle back to this defaults idea in the future. With the encoding issue fixed, I think the results are now better regardless of temperature - may have just been coincidental that low temperature improved the outputs prior to encoding fix.

I'm going to remove this part from the PR for now.

Reply:
Removed.